controllers/users.controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectUsers } = require("../models/users.model");
+
+exports.getUsers = (req, res, next) => {
+  selectUsers()
+    .then((users) => res.status(200).send({ users }))
```

Comment:
I would change the res.status etc from being an implicit return and wrap them in {}, we dont necessarily need to wait for this to resolve